web/client/themes/default/less/widget.less

```diff
@@ -219,6 +219,8 @@
                 .catalog-footer {
                     z-index: 100;
                     .shadow-soft-up;
+                    display: flex;
```

Comment:
Please align the loader to the right, and proper center vertically both the text and the loader.
The height of the footer should not change when loader appears.